.github/workflows/stale-bot.yml

```diff
@@ -27,6 +28,7 @@ jobs:
     steps:
       - uses: actions/stale@v5
         with:
+          ascending: true
```

Comment:
Instead of doing this, can we duplicate the jobs (and have 4 instead of 2). One will run `asc` the other `desc`.

I'm afraid that by doing `asc` with both, we'll lose activity and pings on recent issues (which we don't want to do).

Reply:
ok, let me do that